GETTING_STARTED.md

```diff
@@ -0,0 +1,374 @@
+# Getting started
+This getting started shows you how to use k-ramel with React, but you can use k-ramel with something else (or as a standalone).
+
+But first of all, you have to install dependencies: `yarn add k-ramel @k-ramel/driver-http @k-ramel/react`
+ - **k-ramel** is the base package
+ - **@k-ramel/driver-http** will be used to request some API
+ - **@k-ramel/react** will be used to connect k-ramel with ReactJS
+
+## 1. Create the store
+```diff
++import { createStore, types } from 'k-ramel'
++
++const store = createStore(
++  {
++    todos: types.keyValue(),
++  },
++)
+```
+
+Here we created a store with `todos` that is a keyValue types.
+
+## 2. Use the store
+```diff
+import { createStore, types } from 'k-ramel'
+
+const store = createStore({
+  todos: types.keyValue(),
+})
+
++store.todos.add({ id: 2, title: 'finish the documentation' }) // a.
++store.todos.add({ id: 32, title: 'an other' })
+
++console.log(store.todos.get(2)) // b.
+```
+
+Now you can use the store :
+  - (a) We added a todo, the id is the default key used by keyValue
+  - (b) We retrieve (and log) the todo that has the key equals to `2`
+
+## 3. Provide the store to React (context)
+```diff
++import React from 'react'
+import { createStore, types } from 'k-ramel'
++import { provider } from '@k-ramel/react'
+
+const store = createStore(
+  {
+    todos: types.keyValue(),
+  },
+)
+
+store.todos.add({ id: 2, title: 'finish the documentation' })
+store.todos.add({ id: 32, title: 'an other' })
+
+console.log(store.todos.get(2))
++
++const App = () => (
++  <div>Hey!</div>
++)
++
++export default provider(store)(App)
+```
+
+Now the store is provided in React context, we can use it and bind our a component to it!
```

Comment:
The store is now accessible in the React context

our a component -> a component